specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualNetworkGateway.json

```diff
@@ -1231,6 +1231,288 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/getFailoverAllTestsDetails": {
```

Comment:
>getFailoverAllTestsDetails [](http://example.com/codeflow?start=151&length=26)

Why is this modelled as a POST request?

Reply:
This is an async operation and a long running request, hence added it as POST